doc/sphinx/language/core/inductive.rst

```diff
@@ -1,6 +1,22 @@
 Inductive types and recursive functions
 =======================================
 
+The :cmd:`Inductive` command allows defining types by cases on the form of the
+:term:`inhabitants <inhabitant>` of the type. In contrast, in types defined by the
+:cmd:`Variant` command, constructors can recursively have arguments in
+the type being defined. Inductive types include natural numbers,
```

Comment:
Misworded?  Variants are not recursive (as stated a few sentences below).

Reply:
I think, before, there was "On the contrary of types defined by the ..." but I did not realize that the changes that you proposed changed the meaning. Can I let you do a suggestion?

Reply:
How about this?

```suggestion
:term:`inhabitants <inhabitant>` of the type. These constructors can recursively
have arguments in the type being defined.  In contrast, in types defined by the
:cmd:`Variant` command, such recursive references are not permitted.
Inductive types include natural numbers,
```

Reply:
Yes, I like it.